grpc/execution/server.go

```diff
@@ -336,3 +359,7 @@ func ethHeaderToExecutionBlock(header *types.Header) (*astriaPb.Block, error) {
 		},
 	}, nil
 }
+
+func (s *ExecutionServiceServerV1Alpha2) syncMethodsCalled() bool {
```

Comment:
is this related to the rollup data functionality or was it needed for something else?

Reply:
This isn't needed for rollup functionality, but added to stop us from accidentally executing over bad state in case of something like Geth reboots but conductor didn't and txs/blocks lost.